tokio-util/src/io/copy_to_bytes.rs

```diff
@@ -0,0 +1,68 @@
+use bytes::Bytes;
+use futures_sink::Sink;
+use pin_project_lite::pin_project;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+pin_project! {
+    /// A helper structure which wraps a `Sink<&'a [u8]>` and converts it into
+    /// a `Sink<Bytes>` by copying each byte slice into an owned [`Bytes`].
+    /// This can be convenient when dealing with a [`SinkWriter`] which wraps a sink that
```

Comment:
If this is meant to be a separate paragraph, you'll need a blank line in the docs - Markdown ignores non-empty lines.

Reply:
This was more to adhere to Alice's suggestion to stay within a line length column of 80 signs. No special formatting in mind.